install/ontology/general.rdf

```diff
@@ -0,0 +1,111 @@
+<?xml version="1.0"?>
+<rdf:RDF
+        xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#"
+        xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#"
+        xmlns:lom="http://www.tao.lu/Ontologies/lom.rdf#"
+        xmlns:tao="http://www.tao.lu/Ontologies/TAO.rdf#"
+        xml:base="http://www.tao.lu/Ontologies/lom.rdf#"
+        xmlns:generis="http://www.tao.lu/Ontologies/generis.rdf#"
+        xmlns:widget="http://www.tao.lu/datatypes/WidgetDefinitions.rdf#"
+>
+
+    <!-- LOM properties for General category -->
+    <!-- General fields to define AssessmentContentObject definition -->
+
+    <!--
+      - General identifier -
+      - Identifier value must have an entry that matches the identifier of the associated AssessmentContentObject.
+    -->
+    <rdf:Description rdf:about="http://www.tao.lu/Ontologies/lom.rdf#general-identifier">
+        <rdf:type rdf:resource="http://www.tao.lu/Ontologies/lom.rdf#lomProperty"/>
+        <rdfs:label xml:lang="en-US"><![CDATA[LOM General identifier]]></rdfs:label>
+        <rdfs:comment xml:lang="en-US"><![CDATA[LOM identifier into General category]]></rdfs:comment>
+        <rdfs:domain rdf:resource="http://www.tao.lu/Ontologies/TAO.rdf#AssessmentContentObject"/>
+        <rdfs:range rdf:resource="http://www.w3.org/2000/01/rdf-schema#literal"/>
+        <lom:lomCategoryProperty rdf:resource="http://www.tao.lu/Ontologies/lom.rdf#General"/>
+        <generis:is_language_dependent rdf:resource="http://www.tao.lu/Ontologies/generis.rdf#False"/>
+        <generis:validationRule><![CDATA[notEmpty]]></generis:validationRule>
```

Comment:
Actually, it can be empty in TAO per-se. We cannot ask end users to systematically add an identifier to their assessmentContent resources. At export time, if there is some data for the identifier property, fine, we export it. Otherwise, no identifier exported.